android-agent/src/main/java/io/opentelemetry/android/OpenTelemetryRumBuilder.java

```diff
@@ -377,14 +379,30 @@ private void scheduleDiskTelemetryReader(
         }
     }
 
+    /**
+     * Adds a callback to be invoked after the OpenTelemetry SDK has been initialized. This can be
+     * used to defer some early lifecycle functionality until the working SDK is ready.
+     *
+     * @param callback - A callback that receives the OpenTelemetry SDK instance.
+     * @return this
+     */
+    public OpenTelemetryRumBuilder addOtelSdkReadyListener(Consumer<OpenTelemetrySdk> callback) {
```

Comment:
Consumer is API 24+ only, but I guess its desugared if isCoreLibraryDesugaringEnabled enabled right

Reply:
I think so, yeah...and it's used elsewhere.